en/get-started.md

```diff
@@ -0,0 +1,735 @@
+---
+title: Get Started With TiDB Operator in Kubernetes
+summary: Learn how to deploy TiDB Cluster in TiDB Operator in a Kubernetes cluster.
+category: how-to
+aliases: ['/docs/dev/tidb-in-kubernetes/deploy-tidb-from-kubernetes-dind/', '/docs/dev/tidb-in-kubernetes/deploy-tidb-from-kubernetes-kind/', '/docs/dev/tidb-in-kubernetes/deploy-tidb-from-kubernetes-minikube/']
+---
+
+
+# Get Started with TiDB Operator in Kubernetes
+
+This document explains how to create a simple Kubernetes cluster and use it to do a basic test deployment of TiDB Cluster using TiDB Operator.
+
+> **Warning:**
+>
+> These deployments are for demonstration purposes only. **Do not use** for qualification or in production!
+
+These are the steps this document follows:
+1. [Create a Kubernetes Cluster](#create-a-kubernetes-cluster)
+2. [Deploy TiDB Operator](#deploy-tidb-operator)
+3. [Deploy TiDB Cluster](#deploy-tidb-cluster)
+4. [Connect to TiDB Cluster](#connect-to-tidb)
+
+If you have already created a Kubernetes cluster, you can skip to step 2, [Deploy TiDB Operator](#deploy-tidb-operator).
+
+If you want to do a production-quality deployment, use one of these resources:
+
++ On public cloud:
+    - [Deploy TiDB on AWS EKS](deploy-on-aws-eks.md)
+    - [Deploy TiDB on GCP GKE (beta)](deploy-on-gcp-gke.md)
+    - [Deploy TiDB on Alibaba Cloud ACK](deploy-on-alibaba-cloud.md)
+
++ In an existing Kubernetes cluster:
+    1. Familiarize yourself with [Prerequisites for TiDB in Kubernetes](prerequisites.md)
+    2. Configure the local PV for your Kubernetes cluster to achieve low latency of local storage for TiKV according to [Local PV Configuration](configure-storage-class.md#local-pv-configuration)
+    3. Install TiDB Operator in a Kubernetes cluster according to [Deploy TiDB Operator in Kubernetes](deploy-tidb-operator.md)
+    4. Deploy your TiDB cluster according to [Deploy TiDB in General Kubernetes](deploy-on-general-kubernetes.md)
+
+
+## Create a Kubernetes Cluster
+
+This section covers 3 different ways to create a simple Kubernetes cluster that can be used to test TiDB Cluster running under TiDB Operator. Choose whichever best matches your environment or experience level.
+
+- [Using kind](#create-a-kubernetes-cluster-using-kind) (Kubernetes in Docker)
+- [Using minikube](#create-a-kubernetes-cluster-using-minikube) (Kubernetes running locally in a VM)
+- [Using the Google Cloud Shell](deploy-tidb-from-kubernetes-gke.md) (Kubernetes running in the Google Kubernetes Engine in Google Cloud Platform)
```

Comment:
No procedure for this way in the following sections.

Reply:
Correct, you have to click that link, which will take you to the page with those instructions. I had a lot of trouble figuring out how to properly organize that, so I kind of gave up. If you have a suggestion, please let me know!

Reply:
Maybe better would be to just add a link directly to the Cloud Shell *here* and a little explanation for what is happening.